README.md

```diff
@@ -135,4 +135,5 @@ Versions
 1.6.5 - Make italics visible on Windows.
 
 ### Blah
-- This branch has fixes for Blah
+- This branch has fixes for Blah and the SuperBlah!
+- SuperBlah is the fastest thing in the Universe. You should know that!!!!!!
```

Comment:
Thanks for this change. This is what i was expecting before. The changes look good to me.